schemas/common/survey_metadata.json

```diff
@@ -57,9 +57,37 @@
         },
         "qid": {
           "$ref": "definitions.json#/qid"
+        },
+        "BLOOD_TEST_BARCODE": {
+          "$ref": "definitions.json#/blood_test_barcode"
+        },
+        "SWAB_TEST_BARCODE": {
+          "$ref": "definitions.json#/swab_test_barcode"
+        },
+        "PARTICIPANT_ID": {
+          "$ref": "definitions.json#/participant_id"
+        },
+        "TEST_QUESTIONS": {
+          "$ref": "definitions.json#/test_questions"
+        },
+        "COLLEX_OPEN_DATE": {
+          "$ref": "definitions.json#/collex_open_date"
+        },
+        "COLLEX_CLOSE_DATE": {
+          "$ref": "definitions.json#/collex_close_date"
+        },
+        "FIRST_NAME": {
+          "$ref": "definitions.json#/first_name"
         }
       },
-      "required": ["qid"]
+      "required": [
+        "qid",
+        "PARTICIPANT_ID",
+        "TEST_QUESTIONS",
+        "COLLEX_OPEN_DATE",
+        "COLLEX_CLOSE_DATE",
```

Comment:
Is this marked as required because SDX needs it or because RAS always send it?. If I am not mistaken, I think they are looking to remove the collex date usages from the schema, so if this is required, @Jon-Kent you will need to speak to Author (Martyn) to ensure those are marked as required in the schema JSON otherwise, eQ makes no guarantees that you will receive them.

Reply:
no i don't think SDX needs it, it's just something we always send and i thought was needed for display purposes in EQ, happy to not make it mandatory if you think that makes more sense?

Reply:
See what @Jon-Kent thinks. This only needs to define anything that SDX must receive. As I mentioned above, the above is used by Jon's validation only, as eQ doesn't guarantee these will exist based on the about schema definition. For that, the eQ JSON will need to define them, which sits with Jon to speak with Author about aligning.

Anything eQ needs i.e for display purposes will be marked in eQ JSON as required.

Reply:
ok cool that makes sense, i'll get them removed then 👍

Reply:
done